scripts/sysman/frmwareSecurityVersion.yml

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #----------------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for firmware security version"
+version: "1.9"
+--- #----------------------------------------------------------------------------------
+type: macro
+desc: "Anti roll-back security firmware version"
+version: "1.9"
+name: $S_ANTI_ROLLBACK_SECURITY_VERSION_EXP_NAME
+value: '"$XS_extension_anti_rollback_security_version"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Anti rollback security version Extension Version(s)"
+version: "1.9"
+name: $s_firmware_security_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get anti roll-back security firmware version number of the current running firmware"
+version: "1.9"
+class: $sFirmware
+name: GetSecurityVersionExp
+details:
+    - "The application may call this function from simultaneous threads."
```

Comment:
Do we really need Exp API ? As the API is new and its not conflicting or breaking old API.

Reply:
prefer experimental APIs for now, it has better flexibility. we can always promote them in future as needed once APIs are stable